invenio_oauthclient/alembic/97bbc733896c_create_oauthclient_tables.py

```diff
@@ -73,6 +73,7 @@ def upgrade():
         sa.Column('secret', sa.Text(), nullable=False),
         sa.ForeignKeyConstraint(
             ['id_remote_account'], [u'oauthclient_remoteaccount.id'],
+            name='fk_oauthclient_remote_token_remote_account'
```

Comment:
I agree that this should normally be the case, however I think since no production installations actually uses alembic upgrades yet it is not worth the effort.